subsys/net/lib/fota_download/src/util/fota_download_full_modem.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef FOTA_DOWNLOAD_FULL_MODEM_H__
+#define FOTA_DOWNLOAD_FULL_MODEM_H__
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+int fota_download_full_modem_pre_init(void);
```

Comment:
Should there be some docs for these internal header files?

Reply:
Sure I can add some intro